io.github.lucasew.regex101.yaml

```diff
@@ -0,0 +1,33 @@
+id: io.github.lucasew.regex101
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: 5.15-23.08
+command: io.github.lucasew.regex101
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
it's either x11 alone or wayland + fallback-x11. Not both.
(that's the current linter error)

Reply:
ok